lib/components/molecules/Form/FieldFileInput/tests/FieldFileInput.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { shallow, render } from 'enzyme';
+import { FieldFileInput } from '../FieldFileInput';
+
+describe('FieldInput Component', () => {
```

Comment:
These tests are good start but not enough, ask your self does it cover every line of code you have written in component?